druid-shell/src/backend/windows/window.rs

```diff
@@ -1355,7 +1353,7 @@ impl WindowBuilder {
                 present_strategy: self.present_strategy,
             };
 
-            let (pos_x, pos_y) = match self.position {
+            let (mut pos_x, mut pos_y) = match self.position {
```

Comment:
Don't these also need to be scaled?

Reply:
Sub window position need to be scaled because they are shown in relation to their parent. But for app/parent windows, _mayyyyyybe_ it's OK to now scale them because you wont notice much difference?? I do feel I need to scale these too but left it as-is because it is not related to this tooltip PR.

Reply:
I think it really does need to be scaled, but I'd be happy with a `TODO` comment for now.

Reply:
I'll add a `TODO`.

Reply:
Oops. I said I'd add a `TODO` but I forgot. Sorry. I'll add it now.